core/app/c/[communitySlug]/types/TypeBlock.tsx

```diff
@@ -90,44 +90,52 @@ const TypeBlock: React.FC<Props> = function ({ type, allowEditing }) {
 	};
 	return (
 		<Card>
-			<CardContent className="px-6 py-2">
-				<div className="flex items-center justify-between">
-					<h2 className="flex-grow font-bold">{type.name}</h2>
+			<CardHeader className="flex flex-row items-center justify-between">
+				<h2 className="flex-grow font-bold">
+					{type.name}{" "}
+					<span
+						className="inline-block whitespace-nowrap rounded-md bg-gray-100 p-0.5 text-xs text-gray-500"
+						data-testid={`pubtype-${type.name}-id`}
+					>
+						<pre className="font-mono">{type.id}</pre>
+					</span>
```

Comment:
This was super useful for testing, but I wonder if we should hide the id within some kind of disclosure, or even just add a copy button (similar to the one for copying the token).